src/applications/simple-forms/form-upload/containers/UploadPage.jsx

```diff
@@ -8,12 +8,12 @@ import {
 } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
 import { useHistory, useLocation } from 'react-router-dom';
 import { useDispatch } from 'react-redux';
-import { submitToSimpleForms } from '../actions';
 import {
   getBreadcrumbList,
   getFormNumber,
   getFormUploadContent,
   handleRouteChange,
+  uploadScannedForm,
```

Comment:
I moved this so that I could get rid of the `actions` file. We can do without it.